### PR TITLE
Generator api post with created by

### PR DESCRIPTION
**Pull request by shriramshankar**

### CODE REVIEW

File: api/v1/helpers/verbs/doPut.js [hunk 1]
```diff
@@ -70,9 +70,10 @@ function doPut(req, res, next, props) {
...
             o.changed(key, false);
           } else {
-
-            // value may have changed. set changed to true to
-            // trigger checks in the model
+            /**
+             * value may have changed. set changed to true to
+             * trigger checks in the model
+             */
             o.changed(key, true);
```
Comment (iamigo) [inline api/v1/helpers/verbs/doPut.js]: Is this really necessary here? (I thought we were doing this just for samples and subjects in order to make sure we generated the realtime events.)

Comment (shriramshankar) [inline api/v1/helpers/verbs/doPut.js]: I did not change line 77, just corrected the formatting for the comments.  Need to check why line 77 was added.

Comment (shriramshankar) [inline api/v1/helpers/verbs/doPut.js]: Looks like 77 was added on May 18th to fix the PUT issues from the admin.

**Comment by shriramshankar** [comment]
Merging this PR now.

**Pull request merged**